### DISCUSSION TITLE
3D objects in sketchfab

### DISCUSSION THREAD

**Discussion by jenn1991** [mozilla/hubs#3865] [Spoke Questions] [answered]
Good afternoon
A query a month ago I made a quarter in mozilla and the 3D model worked perfectly but now that it enters the spoke it can no longer be loaded, the model is https://sketchfab.com/3d-models/centro-convenciones-fec74b738014476b8516b7df7f6b16eb.
Suddenly can you help because now it is no longer loaded?

**Comment by j-conrad** [comment, marked as answer]
I wasn't sure if this question got answered elsewhere--perhaps in the Hubs Discord group...

When clicking on 'More Model Information' in Sketchfab, you'll see that the textures on this model are quite large. This file size (over 150mb) could keep it from loading. It's also worth noting that you'll get better performance when using textures whose dimensions are powers of 2 (64, 128, 256, 512, 1024, 2048, 4096).

![image](https://user-images.githubusercontent.com/31968496/109543470-2bd6de00-7a7b-11eb-80be-f65817b24997.png)

**Comment by johnshaughnessy** [comment]
Marked @j-conrad 's response as the answer. Unmark it if you are still having the same issue @jenn1991
